core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -121,6 +121,8 @@
 
   private boolean hasDataMapSchema;
 
+  private Map<String, Object> propertiesMap;
```

Comment:
There is a table property map in the TableSchema already, it is in the TableInfo